ci/build.ps1

```diff
@@ -72,18 +75,20 @@ python  -c "import neovim; print(str(neovim))" ; exitIfFailed
 python3 -c "import neovim; print(str(neovim))" ; exitIfFailed
 
 $env:PATH = "C:\Ruby24\bin;$env:PATH"
-cmd /c gem.cmd install neovim ; exitIfFailed
-where.exe neovim-ruby-host.bat ; exitIfFailed
+gem.cmd install neovim ; exitIfFailed
+Get-Command -CommandType Application neovim-ruby-host.bat
 
-cmd /c npm.cmd install -g neovim ; exitIfFailed
-where.exe neovim-node-host.cmd ; exitIfFailed
-cmd /c npm link neovim
+npm.cmd install -g neovim ; exitIfFailed
+Get-Command -CommandType Application neovim-node-host.cmd
+npm.cmd link neovim
```

Comment:
do we need `; exitIfFailed` here?

Reply:
Not really. Keeping `exitIfFailed` stops the script at the `npm.cmd` command but the build will stop at `Get-Command`.